### ISSUE TITLE
Get UnsupportedOperation: seek Error from idf.run()

### ISSUE THREAD

**Issue by nevr1106** [santoshphilip/eppy#263]
I am using Anaconda python 3.7 distribution with eppy installed from pip. EnergyPlus version is 9.1.0.

Through some testing, I found out that whenever EnergyPlus produces a _severe error_ or requires _preprocessing_, eppy will produce a **UnsupportedOperation: seek** error.

For example, if my EnergyPlus model uses **GroundHeatTransfer:Control** Slab Preprocessor, the eppy idf.run() will terminate and get **UnsupportedOperation: seek** error. This will not occur if the model is run from EnergyPlus itself.

**UnsupportedOperation: seek** will also happen if the EnergyPlus model itself produces a severe error and terminate. 

Here is my code:
```
import sys
pathnameto_eppy = 'F:\Simulation_on_Scale\EppyTest'
sys.path.append(pathnameto_eppy)

from eppy.modeleditor import IDF

iddfile = "C:\EnergyPlusV9-1-0\Energy+.idd"
IDF.setiddname(iddfile)

idfname = './TestIdf/SF_gasfurnace_crawlspace_IECC_2006_AFUE80SEER16.idf'
epwfile = './USA_NY_New.York-J.F.Kennedy.Intl.AP.744860_TMY3.epw'

outdir = './TestIdf'

idf1 = IDF(idfname, epwfile)

idf1.run(output_directory = outdir)
```

Here is the error it produces if any one of the two situation happens:
```
runfile('F:/Simulation_on_Scale/EppyTest/EppyRunTest.py', wdir='F:/Simulation_on_Scale/EppyTest')

C:\EnergyPlusV9-1-0\energyplus.exe --weather F:\Simulation_on_Scale\EppyTest\USA_NY_New.York-J.F.Kennedy.Intl.AP.744860_TMY3.epw --output-directory F:\Simulation_on_Scale\EppyTest\TestIdf --idd C:\EnergyPlusV9-1-0\Energy+.idd F:\Simulation_on_Scale\EppyTest\in.idf

Traceback (most recent call last):

  File "<ipython-input-3-e02248671b12>", line 1, in <module>
    runfile('F:/Simulation_on_Scale/EppyTest/EppyRunTest.py', wdir='F:/Simulation_on_Scale/EppyTest')

  File "C:\Users\pshi.THERMOLIFT\anaconda\lib\site-packages\spyder_kernels\customize\spydercustomize.py", line 827, in runfile
    execfile(filename, namespace)

  File "C:\Users\pshi.THERMOLIFT\anaconda\lib\site-packages\spyder_kernels\customize\spydercustomize.py", line 110, in execfile
    exec(compile(f.read(), filename, 'exec'), namespace)

  File "F:/Simulation_on_Scale/EppyTest/EppyRunTest.py", line 28, in <module>
    idf1.run(output_directory = outdir)

  File "C:\Users\pshi.THERMOLIFT\anaconda\lib\site-packages\eppy\modeleditor.py", line 1014, in run
    run(self, weather=epw, idd=idd, **kwargs)

  File "C:\Users\pshi.THERMOLIFT\anaconda\lib\site-packages\eppy\runner\run_functions.py", line 330, in run
    message = parse_error(output_dir)

  File "C:\Users\pshi.THERMOLIFT\anaconda\lib\site-packages\eppy\runner\run_functions.py", line 343, in parse_error
    sys.stderr.seek(0)

UnsupportedOperation: seek
```

**Comment by santoshphilip** [comment]
@jamiebull1 , is this related to issue #245

**Comment by nevr1106** [comment]
So I resolved the problem at least for my model.

First, the model cannot involve preprocessor like basement or slab preprocessor.

Second, the model itself can't produce any severe error.

If these two conditions are met, the idf.run() method works just fine.

**Comment by jamiebull1** [comment]
Thanks for the update. The seek error seems to happen when handling severe errors, so that makes sense.

I'm not familiar with the preprocessors, so I guess we should also look into fixing or documenting that.

**Comment by santoshphilip** [comment]
this has been fixed in pull request #264
